0000-payment-pointers/0000-payment-pointers.md

```diff
@@ -0,0 +1,126 @@
+
+## Payment Setup Protocols
+
+A payment setup protocol is defined very generically as a protocol for exchanging payment information between a sender and receiver before executing the payment.
+
+An example of a payment setup protocol is the [Simple Payment Setup Protocol](https://interledger.org/rfcs/0009-simple-payment-setup-protocol/) used to setup an Interledger payment.
+
+Payment pointers MUST resolve to an endpoint hosted by a payment receiver at which a payment setup protocol can be initiated. This endpoint is known as the payment setup protocol receiver endpoint.
+
+A payment setup protocol receiver endpoint is identified by an https URL as defined in [RFC7230](https://tools.ietf.org/html/rfc7230#section-2.7.2). It MUST accept HTTP GET requests to initiate the protocol.
+
+Payment setup protocols SHOULD define a custom MIME type for their message definitions and use HTTP content negotitation between the client and endpoint server to determine the payment setup protocol and protocol versions supported by the requesting client and the endpoint server.
+
+## Payment Pointer Syntax
+
+This specification uses the Augmented Backus-Naur Form (ABNF) notation of [RFC2234](https://tools.ietf.org/html/rfc2234) and imports the following rules from [RFC3986](https://tools.ietf.org/html/rfc3986#section-3.3)): _host_ and _path-abempty_.
+
+The syntax of a payment pointer is:
+
+```
+  "$" host path-abempty
+```
+
+## Payment Setup Protocol Receiver Endpoint Resolution
+
+Given a payment pointer of the form `"$" host path-abempty` the payment setup protocol receiver endpoint URL is:
+
+```
+  "https://" host path-abempty
+```
+
+If _path-abempty_ is empty then it is assigned the value `/.well-known/pay`.
+
+Note that this is a restrcited profile of the data allowed in a full https URL. Where the URL syntax supports an _authority_ the payment pointer syntax only supports a _host_ which excludes the _userinfo_ and _port_. The payment pointer syntax also excludes the _query_ and _fragment_ parts that are allowed in the URL syntax.
+
+### Examples
+
+The following payment pointers resolve to the specified endpoint URLS:
+
+```
+$example.com ->                https://example.com/.well-known/pay
+$example.com/invoices/12345 -> https://example.com/invoices/12345
+$bob.example.com ->            https://bob.example.com/.well-known/pay
+$example.com/bob ->            https://example.com/bob
+```
+
+# Payment Initiation
+
+Given a payment pointer for a receiver, a sending client should resolve the https URL of the payment setup protocol receiver endpoint according to the rules defined in this specification.
+
+The client should then initiate an https connection with the endpoint server and perform an http GET request to the endpoint URL.
+
+The cleint should express the protocols it supports by including the media types of the protocol messages in the `Accept` header of the request.
```

Comment:
"protocols" is plural, but I think it should be even more explicit that clients should send multiple types in the Accept header if they support multiple protocols (and we should have an example of what that looks like)

Also, typo: "cleint"